website/content/docs/nia/terraform-modules.mdx

```diff
@@ -33,11 +33,11 @@ See below sections for more information on [defining source input](#source-input
 
 CTS monitors defined objects to provide source input to a task. This source input satisfies the requirements defined by the task's Terraform Module's [input variables](https://www.terraform.io/docs/language/values/variables.html).
 
-A task's source input slightly defers the task's condition even though both monitor defined objects. The task's condition monitors defined objects with a configured criteria. When this criteria is satisfied, the task will trigger.
+A task's source input is slightly different from the task's condition, even though both monitor defined objects. The task's condition monitors defined objects with a configured criteria. When this criteria is satisfied, the task will trigger.
```

Comment:
It's confusing to me that the source input is being compared to the task condition because we haven't defined "task condition" in this topic. Can we assume that readers have read through the other topics in this section and understand the condition concept?

Reply:
At this point, I would guess it's likely that a user has heard of a task condition and should have some understanding of it. If I were to rate CTS topics, the details of source input are a little deeper than condition. Let me know your perspective. If it's confusing, happy to remove this paragraph if it's detracting readers and not adding value.